### PR TITLE
Release 8.4.6

### PR DESCRIPTION
**Pull request by goranalkovic-infinum**
Changes:
- added a couple of icons that were missing
- fixed `LinkEditComponent` having duplicate label because Gutenberg
- rounded off some Gutenberg default controls so it looks neater
- updated RC slider and fixed an offset on the wrapper control

### CODE REVIEW

**Review by iruzevic — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a CHANGELOG](https://keepachangelog.com/).
 
+## [8.4.6] - 2023-09-25
```
Comment (iruzevic) [inline CHANGELOG.md:7]: please add also the changes at the bottom of this page

**Review by iruzevic — CHANGES_REQUESTED** [review]
a small change please

**Review by iruzevic — APPROVED** [review]
❤️

**Review by vedranpetrac — DISMISSED** [review]

**Review by iobrado — APPROVED** [review]

**Review by piqusy — DISMISSED** [review]
How 'bout them lint errors eh' ? 🇨🇦 

Congrats on you first "under 10 files changed" PR 🎖️ 🤣

**Review by iruzevic — APPROVED** [review]

**Review by piqusy — APPROVED** [review]

**Review by iobrado — APPROVED** [review]

**Pull request merged**